Author: Roland Rosenfeld <roland@debian.org>
Description: Move man page from man section 1 to man section 8.

--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -60,7 +60,7 @@ CONF_BASE    = @sysconfdir@
 SBIN_DEST    = @sbindir@
 MAN_DIR      = @mandir@
 MAN_DEST     = $(MAN_DIR)/man1
-MAN_PAGE     = privoxy.1
+MAN_PAGE     = privoxy.8
 SHARE_DEST   = @datadir@
 DOC_DEST     = $(SHARE_DEST)/doc/privoxy
 VAR_DEST     = @localstatedir@
--- a/doc/source/privoxy-man-page.sgml
+++ b/doc/source/privoxy-man-page.sgml
@@ -58,7 +58,7 @@
 </refentryinfo>
 <refmeta>
  <refentrytitle>privoxy</refentrytitle>
- <manvolnum>1</manvolnum>
+ <manvolnum>8</manvolnum>
  <refmiscinfo>
   Privoxy &p-version;<![%p-not-stable;[ &p-status;]]>
  </refmiscinfo>
